Intersect.Editor/Localization/Strings.cs

```diff
@@ -4249,14 +4261,6 @@ public partial struct MapList
 
         public partial struct Mapping
         {
-            public static LocaleDictionary<MapInstanceType, LocalizedString> InstanceTypes = new LocaleDictionary<MapInstanceType, LocalizedString>()
```

Comment:
Changing from a typed key to an int is wrong

Reply:
> Changing from a typed key to an int is wrong

there's no handler for that type, followed up how it's currently handled in other strings :thinking:

Reply:
What do you mean there is no handler for that type? Like no JsonConverter?

The others are all being done wrong, but fixing everything we've done horribly is a huge slog. :)

Reply:
Fixed 👍🏽